sats-dna/src/main/kotlin/com/sats/dna/components/upcomingworkouts/Schedule.kt

```diff
@@ -1,4 +1,4 @@
-package com.sats.dna.components.schedule
+package com.sats.dna.components.upcomingworkouts
```

Comment:
Idk if maybe Schedule deserves a bit of a name change? It is very closely tied to upcoming workouts 🤔